std/installer/mod.ts

```diff
@@ -195,6 +198,13 @@ exit $ret
   await chmod(filePath, 0o755);
 }
 
+/**
+ * Install modules
+ * @param moduleName
```

Comment:
If the plugin doesn't allow omission of useless `@param` without any descriptions I would be 👎 for this.  This is just vertical noise in files of no value.

Reply:
It can be tuned as shown here : https://www.npmjs.com/package/eslint-plugin-jsdoc#require-param-description